lalrpop/src/lexer/nfa/mod.rs

```diff
@@ -360,7 +360,15 @@ impl NFA {
                     self.push_edge(s0, Noop, target);
                 }
                 Ok(s0)
-            }
+            },
+
+            // FIXME: Figure out what these should do
+            Expr::AnyByte => panic!("Unsupported"),
+            Expr::AnyByteNoNL => panic!("Unsupported"),
+            Expr::WordBoundaryAscii => panic!("Unsupported"),
```

Comment:
If you look up above, you'll see that there are existing candidates like `Err(Expr:WordBoundary)`, can you just move these `WordBoundary` cases up there? They return `Err(...)` to signal to the user that it is not supported.